components/brave_rewards/resources/extension/brave_rewards/manifest.json

```diff
@@ -25,6 +25,22 @@
       "256": "img/bat-256.png"
     }
   },
+  "content_scripts": [
+    {
+      "matches": [
+        "https://*.twitter.com/*"
+      ],
+      "css": [
+        "content.css"
+      ],
+      "js": [
+        "out/brave_rewards_panel_content.bundle.js"
+      ]
+    }
+  ],
+  "web_accessible_resources": [
+    "img/bat-16.png"
```

Comment:
Again, suggest we expose as little as possible to the site. Here, we can inline an SVG. Or, if it must be PNG (not ideal since not scalable), then that can be inlined via data: URI.

Reply:
SVG should work I think, I'll investigate.